observation_portal/observations/serializers.py

```diff
@@ -456,6 +457,7 @@ def create(self, validated_data):
             request_group = RequestGroup.objects.create(**validated_data)
             request = Request.objects.create(request_group=request_group)
             observation = Observation.objects.create(request=request, **obs_fields)
+            ConfigurationStatus.objects.create(observation=observation, instrument_name='')
```

Comment:
This might be the answer to my question above about ensuring a configuration status, right?